keyboards/automata02/alisaie/readme.md

```diff
@@ -0,0 +1 @@
+60% Alice style keyboard.
```

Comment:
This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/docs/documentation_templates.md)